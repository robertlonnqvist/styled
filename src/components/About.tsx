import React from "react";
import Nav from "./Nav";

const About = (): JSX.Element => {
  return (
    <div className="container">
      <Nav active="about" />
      <h1>Freelance software developer and architect</h1>
      <p>
        <b>I love writing code</b> and being part of every aspect of the
        software development process. I'm passionate about technologies and
        tools that make great products and increase developer productivity.
        Currently, I work as a software architect and tech lead, supporting
        agile development teams to make the right decisions.
      </p>
    </div>
  );
};

export default About;
