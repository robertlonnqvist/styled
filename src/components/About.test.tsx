import React from "react";
import { render } from "@testing-library/react";
import About from "./About";
import { MemoryRouter } from "react-router-dom";

test("renders About component", () => {
  const { container } = render(
    <MemoryRouter>
      <About />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
