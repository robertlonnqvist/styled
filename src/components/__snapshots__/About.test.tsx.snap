// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About component 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="terminal-nav"
    >
      <header
        class="terminal-logo"
      >
        <div
          class="logo terminal-prompt"
        >
          <a
            class="no-style"
            href="/"
          >
            Robert LÃ¶nnqvist
          </a>
        </div>
      </header>
      <nav
        class="terminal-menu"
      >
        <ul>
          <li>
            <a
              class="menu-item active"
              href="/"
            >
              About
            </a>
          </li>
          <li>
            <a
              class="menu-item"
              href="/cv"
            >
              CV
            </a>
          </li>
          <li>
            <a
              class="menu-item"
              href="https://github.com/robertlonnqvist"
            >
              Code
            </a>
          </li>
          <li>
            <a
              class="menu-item"
              href="http://www.linkedin.com/in/robertlonnqvist"
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <h1>
      Freelance software developer and architect
    </h1>
    <p>
      <b>
        I love writing code
      </b>
       and being part of every aspect of the software development process. I'm passionate about technologies and tools that make great products and increase developer productivity. Currently, I work as a software architect and tech lead, supporting agile development teams to make the right decisions.
    </p>
  </div>
</div>
`;
